tests/Http/CampaignTest.php

```diff
@@ -29,6 +29,7 @@ public function testCreatingACampaign()
             'impact_doc' => 'https://www.google.com',
             'start_date' => $firstCampaignStartDate,
             'end_date' => $firstCampaignEndDate,
+            'group_type_id' => 2,
```

Comment:
To figure out why this test was breaking, I looked into the `storage/logs` to try to find any errors and see that a foreign key constraint fails. We need to create a `GroupType` factory in order to get a valid `group_type_id` value into the database that we can use for this field.

```
[2020-06-09 21:31:18] testing.ERROR: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`rogue_test`.`campaigns`, CONSTRAINT `campaigns_group_type_id_foreign` FOREIGN KEY (`group_type_id`) REFERENCES `group_types` (`id`)
```